examples/platforms/jn5189/openthread-core-jn5189-config.h

```diff
@@ -0,0 +1,195 @@
+/*
+ *  Copyright (c) 2019, The OpenThread Authors.
+ *  Copyright (c) 2019, NXP.
```

Comment:
The only copyright holder should be "The OpenThread Authors"

Reply:
Fixed.